### ISSUE TITLE
Windows support

### ISSUE THREAD

**Issue by asampaleanu** [koding/kite#146]
This project looks really interesting and it's a shame that it doesn't work under Windows. 

In a closed issue, you mentioned that Windows support could be added with some work. Could you possibly summarize and point to where this work would need to be done? If you used MingW, hopefully the amount of work wouldn't be that great.

**Comment by xogeny** [comment]
I agree.  I looked at several microservices/RPC frameworks, and this looked quite good.  For my use case, no Windows support is a killer for me.  I know they say it is simple, but that was almost a year ago.  I hope they choose to add it in the future.

**Comment by asampal** [comment]
Just to be clear, above I was suggesting that if it was made relatively clear where to start, someone other than the project founders might be inclined to give Windows support a go.

**Comment by rjeczalik** [comment]
@asampal: There's is some progress on this already going on thanks to @ppknap - the kite already [builds on Windows](https://github.com/koding/kite/blob/master/appveyor.yml) and tests two subpackages. If you wish join the fun, we could eventually get there faster :-)
